control/controlclient/direct.go

```diff
@@ -38,6 +38,7 @@ import (
 	"tailscale.com/net/dnsfallback"
 	"tailscale.com/net/interfaces"
 	"tailscale.com/net/netns"
+	"tailscale.com/net/portmapper"
```

Comment:
This is quite the dependency edge. Now the controlclient depends on upnp libraries, for instance.

What about a new small leaf package for these bools?

Then both controlclient and portmapper can depend on that third package?